src/contexts/account-context.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useMemo } from 'react';
+import { PropsWithChildren, createContext } from 'react';
+import { Account } from '../models';
+import { AccountService } from '../services';
+
+export interface AccountContextInterface {
+  getCatFacts: () => Promise<any>;
```

Comment:
Also I don't understand why should this be a part of the AccountContext - even for the sake of a boilerplate example.